src/main/resources/static/css/styles.css

```diff
@@ -5,4 +5,8 @@
 
 body {
     background-color: whitesmoke;
+}
+
+.dropdown{
+    -webkit-appearance:none;
```

Comment:
I think this can be deleted again? try again how it looks on mac

Reply:
somehow form doest work without this

Reply:
can I leave it there?

Reply:
If it helps with the dropdown on mac, yes. Maybe add a comment why this is here

Reply:
it was here  these hooks hidden. Now form  works fine also without this code clock. Shall I still remove it?
![Screenshot 2024-09-17 at 09 41 40](https://github.com/user-attachments/assets/f46a1ca5-e6cd-4677-9daf-da809023af65)

Reply:
If the arrows are shown without the code -> remove the code.
If you need the code to show the arrows -> put a comment behind the code that that is its purpose